src/common/libflux/test/conf.c

```diff
@@ -120,9 +133,10 @@ void test_basic (void)
 
     /* Add files
      */
-    create_test_file (dir, "01", path1, sizeof (path1), t1);
-    create_test_file (dir, "02", path2, sizeof (path2), tab2);
-    create_test_file (dir, "03", path3, sizeof (path3), tab3);
+    create_test_file (dir, "01", NULL, path1, sizeof (path1), t1);
+    create_test_file (dir, "02", NULL, path2, sizeof (path2), tab2);
+    create_test_file (dir, "03", NULL, path3, sizeof (path3), tab3);
```

Comment:
instead of NULL extension meaning "toml", perhaps just pass "toml", so its clear which ones are toml and which ones are json.  adjust `create_test_file()` as a result (i think).

Reply:
Yeah, that's a good suggestion.

Reply:
Fixed that and force-pushed.